src/scipp/io/hdf5.py

```diff
@@ -472,41 +472,3 @@ def load_hdf5(filename: Union[str, Path]) -> VariableLike:
 
     with h5py.File(filename, 'r') as f:
         return HDF5IO.read(f)
-
-
-def to_hdf5(obj: VariableLike, filename: Union[str, Path]) -> None:
```

Comment:
As the CI says, you forgot to update the imports. Not sure if API reference also needs updating?
Otherwise, looks good.